planning/behavior_velocity_planner/src/scene_module/detection_area/scene.cpp

```diff
@@ -203,6 +203,68 @@ bool DetectionAreaModule::modifyPathVelocity(PathWithLaneId * path, StopReason *
   return true;
 }
 
+// calc smallest enclosing circle with average O(N) algorithm
+std::pair<lanelet::BasicPoint2d, double> calcSmallestEnclosingCircle(
+  const lanelet::ConstPolygon2d & poly)
+{
+  const double eps = 1e-5;
```

Comment:
@veqcc 
can you add description of what eps means?

Reply:
```suggestion
  // The `eps` is used to avoid precision bugs in circle inclusion checkings.
  // If the value of `eps` is too small, this function doesn't work well. More than 1e-10 is recommended.
  const double eps = 1e-5;
```